go.mod

```diff
@@ -142,6 +142,11 @@ require (
 	xorm.io/xorm v0.8.2
 )
 
+require (
+	github.com/google/subcommands v1.0.1 // indirect
+	golang.org/x/mod v0.5.0 // indirect
+)
+
```

Comment:
Could you merge this with the other block on line 150? Strange that it didn't happen automatically - I think running `go mod tidy` might do the trick here.

Reply:
👍 will do